mods/default/tools.lua

```diff
@@ -494,8 +494,25 @@ minetest.register_tool("default:key", {
 })
 
 minetest.register_craft({
-	type = "cooking",
-	output = "default:gold_ingot",
-	recipe = "default:key",
-	cooktime = 5,
+	type = "fuel",
```

Comment:
Maybe you should put this higher key?

Reply:
What do you mean?
`type = "Fuel",`
?
If so, that would not work.

Reply:
Ah you mean above the key code?
That would be ok but any cleanup is best done later, i have just reviewed this and do not want to check it again.
So, not in this PR please.